### ISSUE TITLE
Can't import from forwarded react-router-dom

### ISSUE THREAD

**Issue by brandongregoryscott** [AndcultureCode/AndcultureCode.JavaScript.React#10]
On v0.0.6 --

When we updated the exports to be `export * from ...` to be more concise, I think this broke my local project setup. I'm not able to reference any of the components provided by `react-router-dom`:

![image](https://user-images.githubusercontent.com/11774799/84966085-93601f80-b0de-11ea-87f7-77b940d78559.png)

Referencing my local repo from the project's package.json seems to be fine, as it has a copy of `react-router-dom` installed in `node_modules`. However, if I copy the `dist` folder somewhere else and reference it there (basically the same file structure that is created installed from npm), it breaks the reference and it can't find those components. 

I'm wondering if we need to switch back to named exports for forwarding vendor components/modules, or if there's a way we can wrap the whole export in an object/namespace so we access anything it provides.

Using named exports, the compiled `index.js` looks more like this:
```
var react_router_dom_1 = require("react-router-dom");
exports.BrowserRouter = react_router_dom_1.BrowserRouter;
exports.Switch = react_router_dom_1.Switch;
```

whereas right now it looks like this:
```
__export(require("react-router-dom"));
```

**Comment by wintondeshong** [comment]
@brandongregoryscott yeah, being this is causing ongoing challenges we can definitely switch back to named exports. It was a nice thought that we could _automatically_ forward, but at the end of the day not a huge deal. It may be a few days before I get to loop back around to updating Core, React and Testing packages for explicitly named exports. Whomever gets to it first I suppose.

**Comment by brandongregoryscott** [comment]
No rush on this, just wanted to raise the issue and get some feedback on it. Thanks for taking a look @wintondeshong!

**Comment by brandongregoryscott** [comment]
I'll tackle this, shouldn't be too hateful!

**Comment by wintondeshong** [comment]
Will bundle this in the release I create later today.

**Comment by brandongregoryscott** [comment]
@wintondeshong  🤦 I thought I had grabbed all of the exports from `react-router-dom`, but it seems they break them up into separate places too. It looks like I missed some pretty important ones from `react-router-dom/index.d.ts`:

```
BrowserRouterProps
BrowserRouter
HashRouterProps
HashRouter
LinkProps
Link
NavLinkProps
NavLink
```

Would you be able to quick add those in when you're prepping the release?

**Comment by brandongregoryscott** [comment]
Published with [v0.0.8](https://github.com/AndcultureCode/AndcultureCode.JavaScript.React/releases/tag/v.0.0.8)
